src/components/ItemDetail/composables/useLike.ts

```diff
@@ -17,15 +17,18 @@ const useLike = (
   isLiked: Ref<boolean>
   toggleLike: () => Promise<void>
   balloonWidth: ComputedRef<number>
+  isDisabled: Ref<boolean>
 } => {
   const { id: meID } = useMe()
   const meStore = useMeStore()
   const toastStore = useToast()
   const { measureText, measureGrid } = useMeasure()
 
   const isLiked = ref(props.likes.some(v => meID.value === v.id))
+  const isDisabled = ref(false)
 
   const toggleLike = async () => {
+    isDisabled.value = !isDisabled.value

```

Comment:
```suggestion
    isDisabled.value = true
```
反転だとワンチャンバグったときに困るのとちょっと読むときに頭を使うので明示しましょう